bridge_ui/src/components/Attest/Source.tsx

```diff
@@ -34,6 +34,7 @@ function Source() {
   const handleSourceChange = useCallback(
     (event) => {
       dispatch(setSourceChain(event.target.value));
+      console.log(event.target.value);
```

Comment:
I don't think I've seen you add a console.log before...
But it's fine by me.

Reply:
thanks for the catches!